fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBStreamAggregationTest.java

```diff
@@ -455,9 +578,9 @@ public RecordQueryPlan build(final boolean useNestedResult) {
             final var groupingKeyValue = RecordConstructorValue.ofUnnamed(groupValues);
             final var aggregateValue = RecordConstructorValue.ofUnnamed(aggregateValues);
             if (useNestedResult) {
-                return RecordQueryStreamingAggregationPlan.ofNested(quantifier, groupingKeyValue, aggregateValue);
+                return RecordQueryStreamingAggregationPlan.ofNested(quantifier, groupingKeyValue, aggregateValue, RecordQueryStreamingAggregationPlan.SerializationMode.TO_NEW);
```

Comment:
We probably want to have a bit more of a mix here. Until we're done rolling this out, we probably need both `TO_NEW` and `TO_OLD` tests, which probably means adding this as a builder property and then setting it where appropriate. (That might mean parameterizing much of the existing tests.)

Reply:
so I added more parameterized tests in `FDBStreamAggregationTest`, do you think we need more in other tests? if that's the case, we'll probably need to put it in `RecordQueryPlanConfiguration`.

Reply:
I think having these be paramaterized on the serialization mode is probably sufficient